src/functionalTest/resources/templates/minimal-age-assessment-appeal-started.json

```diff
@@ -1,6 +1,6 @@
 {
-  "homeOfficeReferenceNumber": "123456789",
-  "homeOfficeDecisionDate": "{$TODAY}",
+  "decisionLetterReferenceNumber": "RP/50052/2022",
```

Comment:
Ageassessment appeals start from AG. May be in next commit you can change the reference number to start with AG.